compose.yaml

```diff
@@ -0,0 +1,8 @@
+services:
+  app:
+    build:
```

Comment:
As I mentioned in the first comments, we don't need the structure for `build`, i.e. no `context` inner key here. `build: .` is enough.